attestation-agent/kbs_protocol/src/client/rcar_client.rs

```diff
@@ -104,21 +104,33 @@ impl KbsClient<Box<dyn EvidenceProvider>> {
         let request = Request {
             version: String::from(KBS_PROTOCOL_VERSION),
             tee,
-            extra_params: String::new(),
+            extra_params: serde_json::Value::String(String::new()),
```

Comment:
It's uncommon to use serde types directly like this. Usually this would be solved by a serde annotation on the struct.

Reply:
so the `kbs-types` for `Request` would assign an empty string by default so that this line could be dropped completely? That would probably makes sense. The change here is what the compiler suggested

Reply:
oh, I see. the kbs-types package exposes a serde_json type on its public types. that's even more unusual. I'll open an issue on virtee/kbs-types. but there's not much we can do here.